pkg/apis/v1alpha1/condition.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+type ConditionType string
+
+const (
+	Ready    ConditionType = "Ready"
+	NotReady ConditionType = "NotReady"
```

Comment:
Ready status has value "True or False", do we still need "NotReady" Status?

Reply:
Indeed, removed `NotReady` as we don't need it anymore, thanks.